src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameViewModel.cs

```diff
@@ -149,7 +150,8 @@ private void OnGetSuggestionsCommandExecute()
             {
                 _suggestionsDropdownTelemetry.DropdownButtonClickTimes += 1;
             }
-
+            _cancellationTokenSource?.Dispose();
+            _cancellationTokenSource = new CancellationTokenSource();
```

Comment:
I suspect you are looking for `CancellationSeries`

Reply:
We can use it later, since the 'CancellationSeries' makes it easy to cancel a prior request while starting a new one